charts/devtron/values.yaml

```diff
@@ -160,21 +184,27 @@ components:
     dbconfig:
       secretName: postgresql-postgresql
       keyName: postgresql-password
+# Change below values for nats
   nats:
+    registry: ""
     image: nats:2.9.3-alpine
-    imagePullPolicy: IfNotPresent
     reloader:
-      image: quay.io/devtron/nats-server-config-reloader:0.6.2
+      image: nats-server-config-reloader:0.6.2
       imagePullPolicy: IfNotPresent
     metrics:
-      image: quay.io/devtron/prometheus-nats-exporter:0.9.0
+      image: prometheus-nats-exporter:0.9.0
       imagePullPolicy: IfNotPresent
+    natsBox:
+      image: nats-box
     serviceMonitor:
       enabled: false
     persistence:
       storage: 5Gi
   migrator:
-    image: "quay.io/devtron/migrator:v4.16.2"
+    registry: ""
+    image: "migrator:v4.16.2"
+    kubectlImage: "kubectl:latest"
+    duplicateChart: devtron-utils:dup-chart-repo-v1.1.0
```

Comment:
change it to duplicateChartImage in both bom and values as well as template

Reply:
Done